lession13_dom_event/exercise-3-productCategory/main.js

```diff
@@ -0,0 +1,32 @@
+const selectCategory = document.getElementById('category')
+selectCategory.addEventListener('change', () => {
+  showProduct(selectCategory)
+})
+
+const showProduct = category => {
+  const message = document.getElementById('showProduct')
+  if(category.value === 'laptop') {
+    message.innerHTML = 
+    `<ul>
+      <li>Macbook</li>
+      <li>Dell</li>
+      <li>Lenovo</li>
+      <li>Asus</li>
+    </ul>`
+  }
+  else if (category.value === 'mobile') {
```

Comment:
đặt ngoặc kiểu gì đây
```suggestion
  } else if (category.value === 'mobile') {
```